packages/twenty-front/src/effect-components/PageChangeEffect.tsx

```diff
@@ -87,6 +88,14 @@ export const PageChangeEffect = () => {
       !isMatchingLocation(AppPath.PlanRequired)
     ) {
       navigate(AppPath.PlanRequired);
+    } else if (
+      onboardingStatus &&
```

Comment:
Works well, we will need to revisit this with the new onboarding flow and logic!